Source/ProgressionSystemRuntime/Public/Components/PSSpotComponent.h

```diff
@@ -34,6 +34,10 @@ class PROGRESSIONSYSTEMRUNTIME_API UPSSpotComponent : public UActorComponent
 	UPROPERTY(BlueprintAssignable, Transient, Category = "C++")
 	FPSSpotComponent OnSpotComponentReady;
 
+	/** Locks the player spot when progression for level achieved */
+	UFUNCTION(BlueprintCallable, Category= "C++", meta=(BlueprintProtected))
```

Comment:
Please remove `BlueprintProtected` since it's not under `protected` anymore

Reply:
done with https://github.com/h4rdmol/ProgressionSystem/pull/35/commits/d8191f815c7ce2b397c17fe87dca0ab53c403833